code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -389,13 +389,30 @@
 
 /mob/living/simple_animal/bot/screwdriver_act(mob/living/user, obj/item/tool)
 	if(!(bot_cover_flags & BOT_COVER_LOCKED))
+		tool.play_tool_sound(src)
```

Comment:
where'd this come from? It's good, just curious

Reply:
I just felt like it'd make sense to give a bit of auditory feedback to opening a bot's panel